src/o365/spfx/commands/project/project-rename.ts

```diff
@@ -174,9 +194,11 @@ class SpfxProjectRenameCommand extends BaseProjectCommand {
       cmd.log(`Renaming SharePoint Framework project to '${args.options.newName}'`);
     }
 
-    filePaths.forEach((filePath) => {
-      replaceFileContent(filePath);
-    });
+    replacePackageJsonContent(path.join(this.projectRootPath, 'package.json'));
+    replaceYoRcJsonContent(path.join(this.projectRootPath, '.yo-rc.json'));
+    replacePackageSolutionJsonContent(path.join(this.projectRootPath, 'config/package-solution.json'));
```

Comment:
Rather than using a fixed `/`, we should specify all segments separately and have Node join the path properly following the separator for the current OS, eg `path.join(this.projectRootPath, 'config', 'package-solution.json')`